doc/glossary.rst

```diff
@@ -129,6 +129,16 @@ If you wish to add a missing term, please
         or resting-state :term:`fMRI`, graph theory measures, surface or
         volume-based statistics, etc.
 
+    Tedana
+        `Tedana`_ is a multi-echo :term:`fMRI` data denoising pipeline.
+        :term:`Tedana` originally came about as a part of the ME-ICA pipeline,
+        although it has since diverged. An important distinction is that
+        the ME-ICA pipeline was written for python v2.7,
+        and was not designed in a way that others could easily
+        examine and improve the underlying algorithms for
+        TE-dependent analysis of multi-echo :term:`fMRI` data, tedana now assumes
+        that you are working with data which has been previously preprocessed.
+
```

Comment:
I think that the description should say a few words on how Tedana works: my understanding is that it is based on a (spatial ?) ICA decomposition, but what makes it specific to ME is not obvious to me.